src/emailLogin/README.md

```diff
@@ -29,16 +29,26 @@ Ideally, this should be done by the designated team account, not with your perso
 
 2. Next, run the following command: `npm run jump:<staging | prod>`. This sets up the port-forwarding service.
 
-3. In a separate terminal, run `runMigration.js` with the following command:
+3. Populate the `repos.txt` file with the list of repos, separated by `,`
```

Comment:
can we use a csv file instead?

Reply:
Switched in https://github.com/isomerpages/isomer-conversion-scripts/pull/71/commits/0ca186c36dbde59eb151ff7180c92935a47184bd!